internal/packer/packer.go

```diff
@@ -951,7 +964,8 @@ func filterGoEnv(env []string) []string {
 	return relevant
 }
 
-func logic(cfg *config.Struct, programName string) error {
+func logic(pack *Pack, programName string) error {
```

Comment:
We can go one step further and define logic() and Main() as methods on the Pack type.

Reply:
Yeah good idea.